Client/Frontend/Browser/Playlist/FontScaling.swift

```diff
@@ -0,0 +1,28 @@
+// Copyright 2021 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import Foundation
+import SwiftUI
+
+struct ScaledFont: ViewModifier {
```

Comment:
Also maybe add a little documentation comment about what this does and when to use it?